### ISSUE TITLE
Cannot use object of type stdClass as array

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-People-Directory#110]
This has shown up a handful of times in the production PHP error logs so far. I haven't dug into a why at all, but it looks like it may just be an object that's being used incorrectly. /cc @philcable 

```
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "NOTICE: PHP message: PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in /var/www/wp-content/plugins/wsu-people-directory/includes/class-wsuwp-person-display.php:304"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "Stack trace:"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#0 /var/www/wp-content/plugins/wsu-people-directory/templates/people.php(22): WSUWP_Person_Display::get_data(Object(stdClass), Array)"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#1 /var/www/wp-content/plugins/wsu-people-directory/includes/class-wsuwp-people-directory-page-template.php(851): include('/var/www/wp-con...')"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#2 /var/www/wordpress/wp-includes/class-wp-hook.php(298): WSUWP_People_Directory_Page_Template->directory_content('')"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#3 /var/www/wordpress/wp-includes/plugin.php(203): WP_Hook->apply_filters('', Array)"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#4 /var/www/wordpress/wp-includes/post-template.php(240): apply_filters('the_content', '')"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#5 /var/www/wp-content/themes/spine/articles/article.php(7): the_content()"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#6 /var/www/wordpress/wp-includes/template.php(690): require('/var/www/wp-con...')"
[01-Sep-2017 22:16:59] WARNING: [pool www] child 22132 said into stderr: "#7 /var/www/wordpress/wp-includes/template.php(647): load_template('/var/www/wp-co..."
```

**Comment by philcable** [comment]
I _think_ this may be due to the bug fixed in #106. We'll see if it still shows up after the next release - fingers crossed that it doesn't.

**Comment by philcable** [comment]
Closing this out on the assumption that it was indeed fixed in #106.
